tests/objects/test_player.py

```diff
@@ -6,6 +6,7 @@
 
 
 def test_roll_for_species():
+    player.species = SPECIES[CharacterSpecies.HUMAN]
```

Comment:
this test was failing for me locally because in the data file I had my player species was not human.  I think this is safe to do but let me know otherwise.

Reply:
I think we could add a line to the `Player.reset` function and reset the species to human to make it technically do a full reset. Then, we have a fixture which calls player.reset before every test

Reply:
Sounds like a good plan.